db/db_impl/db_impl.cc

```diff
@@ -650,7 +637,7 @@ Status DBImpl::CloseHelper() {
       if (!s.ok()) {
         ROCKS_LOG_WARN(
             immutable_db_options_.info_log,
-            "Unable to clear writer for WAL %s with error -- %s",
+            "Unable to Sync WAL file %s with error -- %s",
```

Comment:
Is this correct? It looks like this is the status for `log.ClearWriter()` (I didn't look at what it does).

Reply:
It was the original message - most likely incorrect - will fix it.

Reply:
Fixed